apis/druid/v1alpha1/druid_types.go

```diff
@@ -46,6 +43,9 @@ type DruidSpec struct {
 	// Required: in-container directory to mount with common.runtime.properties
 	CommonConfigMountPath string `json:"commonConfigMountPath"`
 
+	// Optional: If set to false, pvc shall be deleted on deletion of CR
+	VolumeReclaimPolicy bool `json:"volumeReclaimPolicy"`
```

Comment:
thanks, this is great :)

what do you think about having `enablePVCDeletionFinalizer` instead ?  And have separate finalizers for separate things to finalize instead of one `finalizers.druid.apache.org` e.g.
when `enablePVCDeletionFinalizer = TRUE` , then operator would add a finalizer `delete-pvc.finalizers.druid.apache.org`  ... in future we might add more type of finalizers.

also, by default set `enablePVCDeletionFinalizer = TRUE` as druid's data is anyways safe in deep storage.

Reply:
agreed will do that changes